src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -96,15 +97,17 @@
 
     private static Executor blindTrustEverythingExecutor;
 
-    public GBFRequestUtil() {
+    static {
+        initTrust();
```

Comment:
Before this change, even if you were calling a static method, you would have had to new up an object, lest you would see an NPE because various static methods assume that `blindTrustEverythingExecutor` has been initialized.  Now it gets initialized automagically.